schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/errors/UnsupportedSchemaTypeException.java

```diff
@@ -18,7 +18,9 @@
 /**
  * Indicates that schema type is invalid.
  */
-public class UnsupportedSchemaTypeException extends Exception {
+public class UnsupportedSchemaTypeException extends RuntimeException {
```

Comment:
This Exception is not really exposed on client side. Extending RTE is a controversial topic and it has both advantages/disadvantages. My preference is to avid checked exceptions.